backend/hct_mis_api/config/settings.py

```diff
@@ -124,7 +124,6 @@
     "cash_assist_datahub_mis": env.db("DATABASE_URL_HUB_MIS"),
     "cash_assist_datahub_ca": env.db("DATABASE_URL_HUB_CA"),
     "cash_assist_datahub_erp": env.db("DATABASE_URL_HUB_ERP"),
-    "registration_datahub": env.db("DATABASE_URL_HUB_REGISTRATION"),
```

Comment:
this should be removed in the next pr after removing models from registration_datahub